agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4866,6 +4867,23 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
 
     //check if the Request Mode bit is set
     if (request_mode) {
+        //TODO Handle 0 or more then 1 sta in list, currenlty cli steers only 1 client
```

Comment:
It is better to compare `request_mode` to the enum. It will be much more clearer that way.